### DISCUSSION TITLE
Wiring The Frame Components - Please Clarify the SDA Wiring

### DISCUSSION THREAD

**Discussion by daveclifford** [DCC-EX/EX-T3-WiFi#56] [General]
Alex, 

Looking at the Assembly.md wiring page, everything is clear in the wiring, except the wires from the Triple Axis Accelerometer
 to the FireBeetle2. 

Can I assume the SDA and SDC leads are connected to each other as well as 3V3 and GND?

 **(The top-view.jpg picture is on an angle and gives the impression SDA is not wired.)**

I created a document that could be a checklist of sorts to aid the assembler in wiring the WIFI Throttle. It is a text (txt) file, but I had to save it as an md (markup) file extension [GITHUB won't allow txt files to be saved].

[DCC-EX-WIFI-wiring-list.md](https://github.com/DCC-EX/EX-T3-WiFi/files/10699605/DCC-EX-WIFI-wiring-list.md)

Dave

[I'm getting so close, I can taste it!]

**Comment by alex-code** [comment]
I designed the frame so the accelerometer pins will be directly opposite the same ones on the FireBeetle.
All 4 need to be connected. The FireBeetle and accelerometer pins are labelled if you're unsure.

https://github.com/DCC-EX/EX-T3-WiFi/blob/master/docs/assembly.md#accelerometer-optional

**Reply by daveclifford** [reply]
I understand about the pins being opposite each other (nice design choice!).  I asked for clarifcation because the picture I cited was taken from an angle such that it appears that SDA is not wired (the GREEN wire appears to be at SCL on the FireBeetle2.

Dave
